pkg/virt-controller/services/template_test.go

```diff
@@ -979,9 +979,9 @@ var _ = Describe("Template", func() {
 				value, ok := pod.Annotations["k8s.v1.cni.cncf.io/networks"]
 				Expect(ok).To(BeTrue())
 				expectedIfaces := ("[" +
-					"{\"interface\":\"net1\",\"name\":\"default\",\"namespace\":\"default\"}," +
-					"{\"interface\":\"net2\",\"name\":\"test1\",\"namespace\":\"default\"}," +
-					"{\"interface\":\"net3\",\"name\":\"test1\",\"namespace\":\"other-namespace\"}" +
+					"{\"interface\":\"37a8eec1ce1\",\"name\":\"default\",\"namespace\":\"default\"}," +
+					"{\"interface\":\"1b4f0e98519\",\"name\":\"test1\",\"namespace\":\"default\"}," +
+					"{\"interface\":\"49dba5c72f0\",\"name\":\"test1\",\"namespace\":\"other-namespace\"}" +
```

Comment:
Please consider holding the hashed values as consts / variables.
I find it a bit hard to follow the string literals.

Reply:
@orelmisan I started to work on it then realized it takes much more changes, let's do it on a follow-up PR to reduce noise from this one.